package.json

```diff
@@ -109,35 +107,32 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "yarn clean:build && lerna run prebuild --parallel && lerna run build --parallel && lerna run postbuild",
-    "build:docs": "yarn build && lerna run docbuild --scope=@patternfly/react-docs --stream",
-    "build:typecheck": "lerna run build:typecheck --scope=@patternfly/react-core --stream",
-    "build:prdocs": "lerna run pr-build --scope=@patternfly/react-docs --stream",
-    "clean": "yarn clean:build",
-    "clean:build": "shx rm -rf packages/*/dist && shx rm -rf packages/patternfly-?/*/dist",
+    "build": "yarn prebuild:pf4 && lerna run --parallel build --since",
```

Comment:
Clean up pre/post build to only required steps, and only build packages that have changed since last release using lerna's --since tag.